nvtabular/loader/tensorflow.py

```diff
@@ -264,7 +268,8 @@ def _get_device_ctx(self, dev):
         # commenting out since device statements cause
         # RuntimeErrors when exiting if two dataloaders
         # are running at once (e.g. train and validation)
-        yield dev
+        yield tf.device("/GPU:" + str(dev))
```

Comment:
From playing around with this yesterday, it seems like this change isn't needed to make it work.

Reply:
yeah its not needed, but it sets us up correctly for the when the real fix is rolled out by TF and it doesn't hurt at all because dev will always be 0.